app/Livewire/ServerEntry.php

```diff
@@ -25,6 +25,23 @@ class="absolute left-0 top-1 bottom-0 w-1 rounded-lg"
             </div>
 
             <div class="flex-1 dark:bg-gray-800 dark:text-white rounded-lg overflow-hidden p-3">
+                @if($server->egg->image)
+                    <div style="
+                        position: absolute;
+                        inset: 0;
+                        background: url('{{ $server->egg->image }}') right no-repeat;
+                        background-size: contain;
+                        opacity: 0.20;
+                        max-width: 680px;
+                        max-height: 140px;
+                    "></div>
+                    <div style="
```

Comment:
What is this empty div doing?

Reply:
I think it was just left over when i was playing with it.

Removed.